libs/island-ui/core/src/lib/private/useDeprecatedComponent.ts

```diff
@@ -1,9 +1,12 @@
 import { useEffect } from 'react'
 
-const useDeprecatedComponent = (componentName: string) =>
+const useDeprecatedComponent = (
+  componentName: string,
+  customMsg = `${componentName} is deprecated`,
+) =>
   useEffect(() => {
     if (process.env.NODE_ENV === 'development') {
-      console.warn(`${componentName} is deprecated`)
+      console.warn(customMsg)
     }
   }, [componentName])
```

Comment:
Can we remove `componentName` from the dependency array, since we are no longer using it in the `useEffect`?

Reply:
good point, this should probably be customMsg, I'll cahnge it